src/deluge/io/midi/midi_device.cpp

```diff
@@ -88,8 +88,8 @@ void MIDIDevice::dataEntryMessageReceived(ModelStack* modelStack, int32_t channe
 		if (msb < 16) {
 			int32_t zone;
 
-			// Lower zone
-			if (channel == 0) {
+			// Master Channel of Lower zone
+			if (ports[MIDI_DIRECTION_INPUT_TO_DELUGE].mpeLowerZoneLastMemberChannel && channel == 0) {
```

Comment:
If mpeLowerZoneLastMemberChannel is greater than 0 it means the "MPE Lower Zone" is enabled, so then it is true that channel 0 is a master MPE channel

Reply:
Can this just use is master channel?